components/director/hack/jwt_generator.sh

```diff
@@ -10,7 +10,7 @@ function get_token(){
     read -r INTERNAL_TENANT_ID <<< $(get_internal_tenant)
 
     local HEADER=$(echo "{ \"alg\": \"none\", \"typ\": \"JWT\" }" | base64 | tr '/+' '_-' | tr -d '=')
-    local PAYLOAD=$(echo "{ \"scopes\": \"webhook:write formation_template.webhooks:read runtime.webhooks:read application.local_tenant_id:write tenant_subscription:write tenant:write fetch-request.auth:read webhooks.auth:read application.auths:read application.webhooks:read application.application_template:read application_template:write application_template:read application_template.webhooks:read document.fetch_request:read event_spec.fetch_request:read api_spec.fetch_request:read runtime.auths:read integration_system.auths:read bundle.instance_auths:read bundle.instance_auths:read application:read automatic_scenario_assignment:read health_checks:read application:write runtime:write label_definition:write label_definition:read runtime:read tenant:read formation:read formation:write internal_visibility:read formation_template:read formation_template:write formation_constraint:read formation_constraint:write certificate_subject_mapping:read certificate_subject_mapping:write formation.state:write tenant_access:write\", \"tenant\":\"{\\\"consumerTenant\\\":\\\"$INTERNAL_TENANT_ID\\\",\\\"externalTenant\\\":\\\"3e64ebae-38b5-46a0-b1ed-9ccee153a0ae\\\"}\" }" | base64 | tr '/+' '_-' | tr -d '=')
+    local PAYLOAD=$(echo "{ \"scopes\": \"webhook:write formation_template.webhooks:read runtime.webhooks:read application.local_tenant_id:write tenant_subscription:write tenant:write fetch-request.auth:read webhooks.auth:read application.auths:read application.webhooks:read application.application_template:read application_template:write application_template:read application_template.webhooks:read document.fetch_request:read event_spec.fetch_request:read api_spec.fetch_request:read runtime.auths:read integration_system.auths:read bundle.instance_auths:read bundle.instance_auths:read application:read automatic_scenario_assignment:read health_checks:read application:write runtime:write label_definition:write label_definition:read runtime:read tenant:read formation:read formation:write internal_visibility:read formation_template:read formation_template:write formation_constraint:read formation_constraint:write certificate_subject_mapping:read certificate_subject_mapping:write formation.state:write tenant_access:write bundle_instance_auth:write\", \"tenant\":\"{\\\"consumerTenant\\\":\\\"$INTERNAL_TENANT_ID\\\",\\\"externalTenant\\\":\\\"3e64ebae-38b5-46a0-b1ed-9ccee153a0ae\\\"}\" }" | base64 | tr '/+' '_-' | tr -d '=')
```

Comment:
As the payload is now changed, we need to adjust the documentation with the correct `Bearer` token. Generate one using the `jwt_generator.sh` and update it in the Readme file [L28](https://github.com/kyma-incubator/compass/blob/main/components/director/README.md)